### PR TITLE
Feat: Issue #51: Add main tests

### PR DESCRIPTION
**Pull request by alevarn**
Solves the issue #51

### CODE REVIEW

File: src/test/java/decide/MainTest.java [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+        validParams.B_PTS = 1;
+        validParams.C_PTS = 1;
+        validParams.D_PTS = 1;
+        validParams.E_PTS = 1;
+        validParams.F_PTS = 1;
+        validParams.G_PTS = 1;
+        validParams.LENGTH2 = 1.0;
+        validParams.RADIUS2 = 1.0;
+        validParams.AREA2 = 1.0;
+
```
Comment (siberov) [inline src/test/java/decide/MainTest.java:39]: I'm not sure if this is the right place, but maybe validate the parameters as well?

Comment (siberov) [reply src/test/java/decide/MainTest.java:39]: The same goes for the rest of the test cases

**Review by siberov — COMMENTED** [review]
It all looks good to me, just had a though about validation of the parameters.

File: src/test/java/decide/MainTest.java (hunk 1 above)
Comment (alevarn) [reply src/test/java/decide/MainTest.java:39]: I have done that see a little bit below:

`assertThrows(IllegalArgumentException.class, () -> Parameters.verify(validParams, points));`

I verify the parameters in all tests.

**Review by siberov — APPROVED** [review]

File: src/test/java/decide/MainTest.java (hunk 1 above)
Comment (siberov) [reply src/test/java/decide/MainTest.java:39]: Oh 😳 
It's getting a bit late. But you can trust that the rest of the review was meticulous!

**Pull request merged**